rpm/bundle-rust-downstream

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/python3
+
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2021 Red Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+#
+# PYTHON_ARGCOMPLETE_OK
+
+import os
+import sys
+import signal
+import argparse
+import argcomplete
+import shutil
+import toml
+from lib389.cli_base import setup_script_logger
+
+SPECFILE_COMMENT_LINE = 'Bundled cargo crates list'
+START_LINE = f"# {SPECFILE_COMMENT_LINE} - START\n"
```

Comment:
minor, but I'd make this "bigger". Like "##### {SPEC..} - START #####" just so the section is really clearly highlighted.